<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Decrypt</class>
 <widget class="QDialog" name="Decrypt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Decryption</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Out file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outFileName"/>
       </item>
       <item>
        <widget class="QPushButton" name="fileDialogButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="decryptButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Decrypt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="decryptInfo">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
